app/models/concerns/audit_extensions.rb

```diff
@@ -76,6 +76,8 @@ def self.auditable_type_complete_values
     before_save :fix_auditable_type, :ensure_username, :ensure_auditable_and_associated_name, :set_taxonomies
     before_save :filter_encrypted, :if => Proc.new {|audit| audit.audited_changes.present?}
     before_save :filter_passwords, :if => Proc.new {|audit| audit.audited_changes.try(:has_key?, 'password')}
+    after_save :log_audit
+    after_destroy :log_audit
```

Comment:
This isn't needed, since audits shouldn't be deleted (unless through the rake task which won't call the callback anyways)

Reply:
fair enough, I also converted after_save to after_create for the same reason